crates/nargo/src/ops/preprocess.rs

```diff
@@ -42,7 +42,7 @@ pub fn preprocess_contract_function<B: ProofSystemCompiler>(
     include_keys: bool,
     common_reference_string: &[u8],
     func: ContractFunction,
-) -> Result<PreprocessedContractFunction, B::Error> {
+) -> Result<(PreprocessedContractFunction, DebugInfo), B::Error> {
```

Comment:
`preprocess_program` did return a tuple of the preprocessed program and the corresponding debug_info, but it was missing for `preprocess_contract_function`